app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageController.scala

```diff
@@ -1070,16 +1079,18 @@ class SingleDisposalsTriageController @Inject() (
                       )
```

Comment:
https://github.com/hmrc/cgt-property-disposals-frontend/pull/730/files#diff-c012fb5c95b54b45bb4f87ae401cf47fR832

the `yearToDateLiaibility` and `manadatory` evidence fields should be unset for both first and further returns